src/styles/variables.js

```diff
@@ -77,7 +77,8 @@ export default {
     lineHeightNormal: getValueUsingPixelRatio(16, 21),
     lineHeightLarge: getValueUsingPixelRatio(18, 24),
     lineHeightXLarge: getValueUsingPixelRatio(20, 24),
-    inputHeight: getValueUsingPixelRatio(50, 72),
+    inputHeight: getValueUsingPixelRatio(52, 72),
+    inputHeightSmall: 26,
```

Comment:
Minor comment here: our small buttons are 28px tall, so I think we'd want the small inputs to match them.

Reply:
Good call out, thx!